### PR TITLE
Moving from Old wiki - Create Customizing using App and Setup

### PR DESCRIPTION
**Pull request by AnthonyNjuguna**
The wiki is missing from the github project - so moving this to the new documentation area

### CODE REVIEW

**Comment by MarcBruins** [comment]
Thanks a lot AnthonyNuguna. Can you make sure there is a space between the # header instead of #header and is also missing the frontmatter yaml which you can see at the top in the other docs: https://github.com/MvvmCross/MvvmCross/edit/develop/docs/_documentation/fundamentals/data-binding.md and as a final remark the ```c# should be add before there is a code sample to show it properly as code. For more info check this: https://www.mvvmcross.com/documentation/contribute/mvvmcross-docs-style-guide

File: docs/_documentation/fundamentals/Customizing-using-App-and-Setup.md [hunk 1]
```diff
@@ -0,0 +1,537 @@
...
+        list.Add(typeof(SomeTypeFromAdditionalViewModelAssembly2).Assembly);
+        list.Add(typeof(SomeTypeFromAdditionalViewModelAssembly3).Assembly);
+        // ...
+        return list.ToArray();
+    }
+```
+
+**Note:** for WindowsPhone only, if you load Views from any Assembly other than the main one, then you need to add a MvxPhoneViewAttribute to each of those Views in order to help MvvmCross work out the Xaml Url for the View. For example, if the SpecialLoginView is a Xaml page located in the SubViews folder of the MyOrg.Satellite.ExtraViews assembly then you will need to add the attribute: [MvxPhoneView("/MyOrg.Satellite.ExtraView;component/SubViews/BlockedUsers.xaml")]
+
+##Overriding ViewModel Location/Construction
```
Comment (MarcBruins) [inline docs/_documentation/fundamentals/Customizing-using-App-and-Setup.md]: This still needs spacing

File: docs/_documentation/fundamentals/Customizing-using-App-and-Setup.md [hunk 2]
```diff
@@ -0,0 +1,537 @@
...
+* the Setup class in the native UI project - this Setup is a bootstrapper for the MvvmCross system and for your app.
+
+## App.cs
+
+Typically App.cs provides only initialization of:
+
+* simple rule-based IoC registration - e.g.:
+
+```c#
+      CreatableTypes()
```
Comment (MarcBruins) [inline docs/_documentation/fundamentals/Customizing-using-App-and-Setup.md]: There is a default tab space for almost all code examples. Can you remove the first tab? After that it look good to merge

**Comment by MarcBruins** [comment]
Looks good! Thanks a lot!

**Comment by AnthonyNjuguna** [comment]
Thanks.

**Pull request merged**